### PR TITLE
Discard invalid configuration minimumPeriod values

### PR DESCRIPTION
**Pull request by jacogr**
Closes https://github.com/polkadot-js/apps/issues/5910

### CODE REVIEW

File: packages/react-hooks/src/useBlockTime.ts [hunk 1]
```diff
@@ -4,33 +4,45 @@
...
       const blockTime = (
+        // Babe
         a.consts.babe?.expectedBlockTime ||
-        a.consts.difficulty?.targetBlockTime ||
-        a.consts.timestamp?.minimumPeriod.muln(2) ||
-        DEFAULT_TIME
+        // POW, eg. Kulupu
+        a.consts.difficulty?.targetBlockTime || (
+          // Check against threshold to determine value validity
+          a.consts.timestamp?.minimumPeriod.gt(THRESHOLD)
```
Comment (jacogr) [inline packages/react-hooks/src/useBlockTime.ts:35]:
```suggestion
          a.consts.timestamp?.minimumPeriod.gte(THRESHOLD)
```

**Comment by joelamouche** [comment]
Thanks :)

File: packages/react-hooks/src/useBlockTime.ts [hunk 2]
```diff
@@ -4,33 +4,45 @@
...
 import { useTranslation } from './translate';
 import { useApi } from './useApi';
 
 type Result = [number, string, Time];
 
-const DEFAULT_TIME = new BN(6000);
+const DEFAULT_TIME = new BN(6_000);
 
-export function useBlockTime (blocks = BN_ONE, apiOverride?: ApiPromise | null): Result {
+// Some chains incorrectly use these, i.e. it is se to values such as 0 or even 2
```
Comment (JoshOrndorff) [inline packages/react-hooks/src/useBlockTime.ts:18]: Thank you for fixing Moonbeam;s issue.

FWIW, I don't think our use is incorrect. The field is called `MinimumPeriod` not `ExpectedPeriod` or `HalfOfTheExpectedPeriod`, and the docs also say `Beware that this is different to the *expected* period
		/// that the block production apparatus provides.`

**Comment by crystalin** [comment]
Awesome @jacogr , that clearly improves our user experience on PolkadotJs. I still wonder why we never thought about it before :)

**Pull request merged**